### PR TITLE
Anim cards

### PR DESCRIPTION
**Pull request by AmauBe**
Cards Animation done in desktop and mobile on artists cards page. Responsive with different effects (hover and active).

### CODE REVIEW

File: src/View/Artist/index.html.twig [hunk 1]
```diff
@@ -11,11 +11,17 @@
 		<div id="allArtists" class="container marketing">
 			<div class="row">
 				{% for artist in artists %}
-					<div id="oneArtist" class="col-6 col-lg-4 col-xl-3">
-						<img src="{{ artist.image_artist }}" class="bd-placeholder-img rounded-circle shadow-lg" role="img" preserveaspectratio="xMidYMid slice" focusable="false">
+					<div id="oneArtist" class="artist position-relative col-6 col-lg-4 col-xl-3">
+						<img src="{{ artist.image }}" class="d-flex position-relative bd-placeholder-img rounded-circle shadow-lg border" role="img" preserveaspectratio="xMidYMid slice" focusable="false" class="info">
```
Comment (Dldvlpr) [inline src/View/Artist/index.html.twig]: You got two class="" in this line.

**Review by Dldvlpr — CHANGES_REQUESTED** [review]

**Comment by AmauBe** [comment]
Problem line 15, correction  done. Normally all is ok on animCards branch.

File: src/View/Artist/show.html.twig [hunk 2]
```diff
@@ -11,9 +11,13 @@
 					<img src="{{ artist.image_artist }}" class="bd-placeholder-img rounded-circle shadow-lg" role="img" preserveaspectratio="xMidYMid slice" focusable="false">
 						<rect fill="#777"></rect>
 					</img>
+<<<<<<< HEAD
```
Comment (Dldvlpr) [inline src/View/Artist/show.html.twig]: Dead code

File: src/View/Artist/show.html.twig [hunk 3]
```diff
@@ -11,9 +11,13 @@
...
+					<p class="artist-name">{{ artist.name }}</p>
+=======
```
Comment (Dldvlpr) [inline src/View/Artist/show.html.twig]: Here too.

**Review by Dldvlpr — CHANGES_REQUESTED** [review]

**Comment by AmauBe** [comment]
Problem rectified on show.html.twig. Dead code on this files deleted.

File: public/assets/css/style.css [hunk 4]
```diff
@@ -98,7 +138,20 @@ a.btn.btn-success.btn-sm {
...
+
+#oneArtist {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin: 1rem 0 1rem 0;
+}
+=======
 /*************** HEADING **********************/
+>>>>>>> dev
```
Comment (aldaitzwild) [inline public/assets/css/style.css]: Unresolved conflicts !

**Review by aldaitzwild — CHANGES_REQUESTED** [review]

File: src/View/Artist/show.html.twig [hunk 5]
```diff
@@ -1,7 +1,7 @@
 {% extends 'layout.html.twig' %}
 
 {% block content %}
-	<main class="container">
+	<main class="containerart">
```
Comment (aldaitzwild) [inline src/View/Artist/show.html.twig]: This class does not exist

**Review by aldaitzwild — CHANGES_REQUESTED** [review]

**Comment by AmauBe** [comment]
Correction on show twig files about class="containerart" witch not existed. Remplace by class="container".

File: src/View/layout.html.twig [hunk 6]
```diff
@@ -10,7 +10,7 @@
 		</title>
 		<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-Zenh87qX5JnK2Jl0vWa8Ck2rdkQ2Bzep5IDxbcnCeuOxjzrPF/et3URy9Bv1WTRi" crossorigin="anonymous">
 		<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/js/bootstrap.bundle.min.js" integrity="sha384-OERcA2EqjJCMA+/3y+gxIOqMEjwtxJY7qPCqsdltbNJuaOe923+mo//f6V8Qbsw3" crossorigin="anonymous"></script>
-		<link rel="stylesheet" href="/assets/css/style.css"> {% block stylesheet %}{% endblock %}
+		<link rel="stylesheet" href="assets/css/style.css"> {% block stylesheet %}{% endblock %}
```
Comment (aldaitzwild) [inline src/View/layout.html.twig]: No the "/" before assets is important here

**Review by aldaitzwild — CHANGES_REQUESTED** [review]

**Comment by AmauBe** [comment]
Error "/" rectified on layout. "/" put on asset on layout.html.twig

**Review by aldaitzwild — APPROVED** [review]

**Pull request merged**